package.json

```diff
@@ -8,6 +8,7 @@
     "start:catalog": "turbo start --filter pagopa-interop-catalog-process",
     "start:catalog-consumer": "turbo start --filter pagopa-interop-catalog-event-consumer",
     "start:agreement": "turbo start --filter pagopa-interop-agreement-process",
+    "start:agreement-consumer": "turbo start --filter pagopa-interop-agreement-event-consumer",
```

Comment:
I agree the name maybe too generic, and perhaps it would be more intuitive to use the postfix `sink` to indicate this kind of services, I would omit `readmodel` which is more of an implementation detail, however a naming of the services refactor is onerous because there are really many nomenclatures to change 😅. 
I propose to discuss it and then possibly manage it in a separate PR.

Reply:
`readmodel` is the actual and only purpose of this consumer 😄 

Fine by me to do this in a separated PR, but both consumer must be renamed